static/js/plotjs/stackf.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable */
+const binsize = slider.value;
+const fluxalph = ((binsize === 0) ? 1.0 : 0.1);
+
+for (let i = 0; i < toggle.labels.length; i++) {
+  const fluxsource = eval(`obs${i}`).data_source;
+  const binsource = eval(`bin${i}`).data_source;
+
+  const fluxerrsource = eval(`obserr${i}`).data_source;
+  const binerrsource = eval(`binerr${i}`).data_source;
+
+  const minmjd = Math.min.apply(Math, fluxsource.data.mjd) - 15;
```

Comment:
See previous review comment on the suggested zoom windows for t and flux/mag.

Reply:
This is just for stacking windows, not for zoom windows